docker/clickhouse-builder/assets/arm64.compile.Dockerfile

```diff
@@ -25,14 +25,14 @@ ENV LC_ALL en_US.UTF-8
 ENV TZ UTC
 
 RUN apt-get -y install git cmake python ninja-build
-RUN git clone --branch $GIT_TAG --recursive https://github.com/ClickHouse/ClickHouse.git
+RUN git clone --depth 1 --branch $GIT_TAG --recursive https://github.com/ClickHouse/ClickHouse.git
```

Comment:
Is it worth adding `--shallow-submodules` as well?  See https://git-scm.com/docs/git-clone#Documentation/git-clone.txt---depthltdepthgt